base/REPLCompletions.jl

```diff
@@ -167,8 +167,14 @@ function complete_path(path::AbstractString, pos; use_envpath=false)
             for file in filesinpath
                 # In a perfect world, we would filter on whether the file is executable
                 # here, or even on whether the current user can execute the file in question.
-                if startswith(file, prefix) && isfile(joinpath(pathdir, file))
-                    push!(matches, file)
+                if startswith(file, prefix)
+                    try
+                        # isfile can cause an error when trying to ask for stat on a
+                        # system file, it do not have permision to read.
```

Comment:
"it does not"

this should check for the specific known exception case in a catch and rethrow anything else that might happen - otherwise this could swallow serious bugs